config/site_settings.yml

```diff
@@ -3046,6 +3046,9 @@ api:
   retain_web_hook_events_period_days:
     default: 30
     max: 36500
+  retain_web_hook_events_aggregate_days:
+    default: 720
+    max: 36500
```

Comment:
It's pretty comical that we allow such a huge max with this and the other one. No need to change, but it's funny that this ends up being Sunday 28 May 2124 :joy: